android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java

```diff
@@ -57,9 +61,23 @@ public RNPushNotification(ReactApplicationContext reactContext) {
         registerNotificationsRegistration();
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
-            NotificationChannel channel = new NotificationChannel(CHANNEL_ID, CHANNEL_NAME, CHANNEL_IMPORTANCE);
             NotificationManager manager =
                 (NotificationManager) reactContext.getSystemService(reactContext.NOTIFICATION_SERVICE);
+            // Delete old notification channel
+            manager.deleteNotificationChannel(OLD_CHANNEL_ID);
```

Comment:
so you tested that this is fine if this doens't exist right

Reply:
yep, it worked on a fresh install